analysis_ready_planetscope/true_color_cloud_masked/script.js

```diff
@@ -0,0 +1,22 @@
+//VERSION=3
+
+function setup() {
+    return {
+        input: ["blue", "green", "red", "cloud_mask"],
+        output: {
+            bands: 4
+        }
+    };
+}
+
+let factor = 1/2000; 
+function evaluatePixel(sample) {
+    let opacity = sample.cloud_mask == 1 ? 1 : 0;
+
+    return [
```

Comment:
@prashansa Since opacity has been set to 1 or 0 depends on the cloud_mask, it's going to be transparent wherever `cloud_mask == 0`. I think we can return like this. Could you please give it a try if the script produce your expected result?

Reply:
yes, indeed, this makes sense. thank you for catching this! it works and we can merge this change.